bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js

```diff
@@ -328,20 +328,45 @@ class MapModuleOlCesium extends MapModuleOl {
         return params;
     }
 
+    /**
+     * Returns color expressions modified with layer opacity
+     * @param {String | Object} colorDef Cesium style expression or object containing conditions array 
+     * @param {Number} opacity Layer opacity
+     */
+    _getColorExpressionsWithOpacity (colorDef, opacity) {
+        if (!colorDef || isNaN(opacity)) {
```

Comment:
If `colorDef` is falsy, we return it anyway?

Reply:
Yes. If we can't modify the expression, we return it as is.